include/dbg_macro/dbg.h

```diff
@@ -29,23 +29,30 @@ License (MIT):
 #ifndef DBG_MACRO_DBG_H
 #define DBG_MACRO_DBG_H
 
-#pragma message("WARNING: the 'dbg.h' header is included in your code base")
-
+#include <algorithm>
 #include <ios>
 #include <iostream>
 #include <sstream>
 #include <string>
 #include <type_traits>
 #include <vector>
 
-#include <unistd.h>
-
 #if __cplusplus >= 201703L
 #include <optional>
 #endif
 
 namespace dbg_macro {
 
+#if defined(__linux__) || defined(__APPLE__)
```

Comment:
instead of defining two new macros, I would prefer something like this:
``` c++
bool is_colorized_output_enabled() {
#if …UNIX…
  // ...
#elif  …WINDOWS…
  //
#endif
}
```

Reply:
Also, I think we don't have to be linux + apple specific here. Every POSIX system should also come with `<unistd.h>`:
``` c++
#if defined(__unix__) || defined(__APPLE__)
```

Reply:
OK, Just give me some time I need to take a break!

Reply:
Done! Please let me know if that's what you wanted.